contracts/src/CrossChainApplications/ERC20Bridge/tests/ERC20BridgeTests.t.sol

```diff
@@ -609,6 +624,72 @@ contract ERC20BridgeTest is Test {
         assertEq(address(mockERC20), newBridgeToken.nativeAsset());
     }
 
+    function testZeroTeleporterRegistryAddress() public {
+        vm.expectRevert(
+            "TeleporterUpgradeable: zero teleporter registry address"
+        );
+        new ERC20Bridge(address(0));
+    }
+
+    function testUpdateMinTeleporterVersion() public {
+        // Check that updating minimum Teleporter version fails if it's not the contract owner.
+        vm.prank(address(0));
+        vm.expectRevert("Ownable: caller is not the owner");
+        erc20Bridge.updateMinTeleporterVersion();
+
+        // Check that the owner can update the minimum Teleporter version.
+        vm.mockCall(
+            MOCK_TELEPORTER_REGISTRY_ADDRESS,
+            abi.encodeWithSelector(
+                WarpProtocolRegistry.getLatestVersion.selector
+            ),
+            abi.encode(2)
+        );
+        vm.expectCall(
+            MOCK_TELEPORTER_REGISTRY_ADDRESS,
+            abi.encodeWithSelector(
+                WarpProtocolRegistry.getLatestVersion.selector
+            )
+        );
+        erc20Bridge.updateMinTeleporterVersion();
```

Comment:
Can we `expectEmit` of the specific event here, and also check the value of the minimum version is the expected value afterwards?